src/features/Courses/CoursesTable/columns.jsx

```diff
@@ -30,6 +37,45 @@ const columns = [
     Header: 'Students invited',
     accessor: 'numberOfPendingStudents',
   },
+  {
+    Header: '',
+    accessor: 'className',
+    cellClassName: 'dropdownColumn',
+    Cell: ({ row }) => {
+      const [isOpenModal, openModal, closeModal] = useToggle(false);
+      const courseDetailsLink = `${getConfig().LEARNING_MICROFRONTEND_URL}/course/${row.original.masterCourseId}/home`;
+      return (
+        <Dropdown className="dropdowntpz">
```

Comment:
This happens to me when I only have 2 courses, the modal is being cover, maybe a little bit more `z-index` is necessary. 

![ee](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/7530240a-9644-4c9a-b57c-5996fac809f0)